### PR TITLE
nixos/release{,-combined}: drop the VirtualBox OVA

### PR DESCRIPTION
**Pull request by emilazy**
VirtualBox has been broken for over a week on `staging-next`.

See https://github.com/NixOS/nixos-channel-scripts/pull/75.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by emilazy** [comment]
cc @FriedrichAltheide @svanderburg @blitz

**Comment by SuperSandro2000** [comment]
Does that mean the iso and the ova on the website then no longer match?

**Comment by vcunat** [comment]
What do you mean?  [The website](https://nixos.org/download/#nixos-virtualbox) points to 24.05, but that's not touched here.

**Comment by blitz** [comment]
> cc @FriedrichAltheide @svanderburg @blitz

I'm fine with dropping the OVA. The ISO should work fine.

**Comment by emilazy** [comment]
After discussion with @blitz, I’ve updated this PR to drop the OVA from the channel entirely, in line with the recent direction of trying to move toward building fewer images in the release channels in general, as it is explicitly marked on the website as only being suitable for demo purposes, does not really add meaningful value in that role over the standard ISO that most users will download, and takes up 2½ gigabytes of cache space whenever the installer changes that can be better spent on other things.

Hopefully we can move forward with this given the VirtualBox maintainer sign‐off and help reduce image‐related cache size bloat. The channel scripts PR will need merging and deploying before this one; if they’re merged I will prepare a PR for the website to remove the OVA link for 24.11.

**Review by mweinelt — APPROVED** [review]

**Review by RossComputerGuy — APPROVED** [review]

**Comment by emilazy** [comment]
Will open a draft homepage PR hopefully later today. `nixos/modules/installer/virtualbox-demo.nix` is still around unreferenced and can be removed if it bothers anyone; it will probably stop evaluating at some point.

**Pull request merged**
